src/index.native.js

```diff
@@ -110,8 +113,20 @@ class Screen extends React.Component {
 
       // When using RN from master version is 0.0.0
       if (version.minor >= 57 || version.minor === 0) {
-        const { enabled, ...rest } = this.props;
-        return <AnimatedNativeScreen {...rest} ref={this.setRef} />;
+        let { enabled, active, activityState, ...rest } = this.props;
+        if (active !== undefined && activityState === undefined) {
+          console.warn(
+            'You are using old React Navigation version. Update your React Navigation navigators to use the best working solution.'
```

Comment:
I'd structure this message along these lines:
Appears like you are using old version of react-navigation library. Please update react-navigation to version X.Y.Z or above to take full advantage of new functionality added to react-native-screens